libs/openFrameworks/sound/ofRtAudioSoundStream.h

```diff
@@ -5,6 +5,7 @@
 #include "ofBaseSoundStream.h"
 #include "ofTypes.h"
 #include "ofSoundBuffer.h"
+#include "RtAudio.h"
 
 class RtAudio;
```

Comment:
I moved this over to the .h as it was originally in the cpp file, removing it makes accessing the RtAudio::Api enum throw an error